src/main/java/uk/gov/hmcts/et/common/model/ccd/CaseData.java

```diff
@@ -1124,4 +1114,43 @@ public class CaseData extends Et1CaseData {
     private String tseAdminCloseApplicationYes;
     @JsonProperty("tseAdminCloseApplicationText")
     private String tseAdminCloseApplicationText;
+
+    //sendNotification
+    @JsonProperty("sendNotificationCollection")
+    private List<SendNotificationTypeItem> sendNotificationCollection;
+    @JsonProperty("sendNotificationTitle")
```

Comment:
Why not replace all of these with your new SendNotificationType

Reply:
This would work, but it would mean a big rewrite of our configs (theres a ton of intricate conditionals that would need porting over). Given the time constraints and blockers on this ticket I believe we should leave as-is. But this is a better approach for future tickets